hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java

```diff
@@ -82,6 +77,12 @@
 import org.apache.hbase.thirdparty.com.google.common.annotations.VisibleForTesting;
 import org.apache.hbase.thirdparty.com.google.common.base.Throwables;
 
+import org.apache.hadoop.hbase.shaded.protobuf.ProtobufUtil;
+import org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos;
+import org.apache.hadoop.hbase.shaded.protobuf.generated.MultiRowMutationProtos.MultiRowMutationService;
+import org.apache.hadoop.hbase.shaded.protobuf.generated.MultiRowMutationProtos.MutateRowsRequest;
+import org.apache.hadoop.hbase.shaded.protobuf.generated.MultiRowMutationProtos.MutateRowsResponse;
```

Comment:
We have multiMutate in this class where we need to make use of MultiRowMutationEndpoint, it is a CPEP so we have to make use of com.google.protobuf instead of the shaded protobuf before this patch.